internal/errs/errs.go

```diff
@@ -258,3 +301,14 @@ func Unpack(err error) []error {
 	}
 	return result
 }
+
+// IsUserFacing identifies whether this error was curated for end-users and
+// returns that error. This is NOT the same as an error that is localized.
+func IsUserFacing(err error) (error, bool) {
+	var userFacingError UserFacingError
+	if errors.As(err, &userFacingError) {
+		return userFacingError, true
+	}
+
+	return err, false
```

Comment:
```suggestion
func IsUserFacing(err error) bool {
	var userFacingError UserFacingError
	if errors.As(err, &userFacingError) {
		return true
	}

	return false
```

If you want to cast the error type you should just use `errors.As()` directly.

Reply:
Fair enough, though for our current use case, this change makes the function pointless. I'm just going to remove it and use `errors.As()` where we were calling it previously